src/components/v5/common/ActionSidebar/partials/Motions/steps/OutcomeStep/partials/VoteStatuses/types.ts

```diff
@@ -0,0 +1,5 @@
+import { VoteStatuses } from '../../types';
+
+export interface VoteStatusesProps {
+  list: VoteStatuses[];
```

Comment:
list -> items

Also, the type for the item should be defined here, and not imported from other place